core/shared/src/main/scala/monocle/syntax/All.scala

```diff
@@ -2,4 +2,4 @@ package monocle.syntax
 
 object all extends Syntaxes
 
-trait Syntaxes extends ApplySyntax with FocusSyntax with FieldsSyntax
+trait Syntaxes extends ApplySyntax with FieldsSyntax
```

Comment:
I just realised that the extension method for partially applying foucs was not in FocusSyntax. It is a mistake.

We need to add to test to show that users can do `user.focus(_.name).get` with `import monocle.syntax.all._`